Scripts/Core/EditorVR.Viewer.cs

```diff
@@ -0,0 +1,83 @@
+#if UNITY_EDITORVR
```

Comment:
I think we should call the file/class PlayArea, not Viewer. ViewerPivot sort of makes sense, but I think Play Area is more explicit about what you use it for.